tests/framework/common_apps.go

```diff
@@ -38,8 +38,15 @@ const (
 
 	// MemRSSPanel is the ID of the MemRSS panel on OSM's MeshDetails dashboard
 	MemRSSPanel int = 13
+
 	// CPUPanel is the ID of the CPU panel on OSM's MeshDetails dashboard
 	CPUPanel int = 14
+
+	// AppProtocolHTTP is the HTTP application protocol
+	AppProtocolHTTP = "http"
+
+	// AppProtocolTCP is the TCP application protocol
+	AppProtocolTCP = "tcp"
```

Comment:
Would just be weird exporting stuff in `pkg/envoy` to an e2e test. This could very well be hardcoded to `http` inline in the e2e test code as well, considering its just a property of the test and is user driven. User can put whatever they choose, although the controller only support http and tcp at the moment.